masterlist.yaml

```
@@ -16283,6 +16283,29 @@ plugins:
         util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 21
 
+  - name: 'The Lost Spires - Enhanced Tomb.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/51414/'
+        name: 'The Lost Spires - Enhanced Tomb'
+    msg:
+      - <<: *renameXtoY
+        subs:
+          - '**meshes\skeleton**'
+          - '**meshes\stroti\skeleton**'
+        condition: 'file("meshes/skeleton") and not file("meshes/stroti/skeleton")'
```

`moveXFromYToZ` would be more appropriate here. Also, since the name of the folder, `skeleton`, is pretty generic, there's a decent chance for other mods to install files to this location & complicate the message. Since these resources are only used in `ASSkeletonAeonIV "Skeleton of Aeon IV" [CONT:xx001626]` & only in the form of the mesh `Stroti\skeleton\skelet01.nif`, we can just instruct users to move it:

```suggestion
      - <<: *moveXFromYToZ
        subs:
          - '**skelet01.nif**'
          - '**meshes\skeleton**'
          - '**meshes\Stroti\skeleton**'
        condition: 'file("meshes/skeleton/skelet01.nif") and not file("meshes/stroti/skeleton/skelet01.nif")'
```